_posts/2024-03-20-strimzi-kraft-migration.md

```diff
@@ -0,0 +1,260 @@
+---
+layout: post
+title:  "Migrate your Strimzi-operated cluster from ZooKeeper to KRaft"
+date: 2024-03-20
+author: paolo_patierno
+---
+
+In the previous [blog post](TBD) we explored how Apache ZooKeeper is used to store Apache Kafka clusters metadata and what are the current limitations.
+We introduced the new KRaft protocol which allows to overcome such limitations and store metadata in Kafka itself, taking into account that the ZooKeeper support will be removed soon.
+For this reason, there is the need to migrate existing clusters from using ZooKeeper to KRaft for storing metadata.
+We also covered the overall procedure to do that.
```

Comment:
This looks a bit odd to me
Maybe something like
`The overall procedure was also covered.`

or maybe connect it to the previous sentence

```
For this reason, there is the need to migrate existing clusters from using ZooKeeper to KRaft for storing metadata, where the overall procedure was also covered.
```

but maybe it's just me 🤷